src/platforms/platform.dom.js

```diff
@@ -92,6 +92,43 @@ module.exports = function(Chart) {
 		return canvas;
 	}
 
+	// Detects support for the passive option to addEventListener
+	// https://github.com/chartjs/Chart.js/issues/4287
+	// https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Safely_detecting_option_support
+	var supportsPassiveOption = (function() {
+		var supports = false;
+		try {
+			window.addEventListener('test', null, Object.defineProperty({}, 'passive', {
+				get: function() {
+					supports = true;
+				}
+			}));
+		} catch (e) {
+			// continue regardless of error
+		}
+		return supports;
+	}());
+
+	function addEventListener(node, type, listener) {
+		if (node.addEventListener) {
```

Comment:
@etimberg `addEventListener` and `removeEventListener` seems supported by [IE9+ and all other browsers](https://caniuse.com/#search=addEventListener). What do you think to simplify these methods and remove `attach/detachEvent` + `['on' + type] =`?